modules/candidate_parameters/jsx/CandidateDOD.js

```diff
@@ -0,0 +1,183 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'Loader';
+
+class CandidateDOD extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: [],
+      formData: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.setFormData = this.setFormData.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({data: data, formData: data}))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  setFormData(formElement, value) {
+    let formData = this.state.formData;
+    formData[formElement] = value;
+    this.setState({
+      formData: formData,
+    });
+  }
+
+  render() {
+    if (this.state.error) {
+      return <h3>An error occured while loading the page.</h3>;
+    }
+
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+
+    let disabled = true;
+    let updateButton = null;
+    if (loris.userHasPermission('candidate_dod_edit')) {
+      disabled = false;
+      updateButton = <ButtonElement label="Update"/>;
+    }
+
+    return (
+      <div className='row'>
+        <FormElement
+          name='candidateDOD'
+          onSubmit={this.handleSubmit}
+          ref='form'
+          class='col-md-6'>
+          <StaticElement
+            label='PSCID'
+            text={this.state.data.pscid}
+          />
+          <StaticElement
+            label='DCCID'
+            text={this.state.data.candID}
+          />
+          <StaticElement
+            label='Disclaimer:'
+            text='Any changes to the date of death requires an
+            administrator to run the fix_candidate_age script.'
+            // class='form-control-static text-danger bg-danger col-sm-10'
```

Comment:
commented out code

Reply:
uncommenting - but should be noted that this only works if the changes in  #4915 are accepted and merged 
specifically this part:
https://github.com/aces/Loris/pull/4915/files#diff-bb24eb9a9e1227db8dd174deb9712093R1445

Reply:
Should this PR be marked as Blocked then?